File: blink.py
import mraa
import time
import sys

x = mraa.Gpio(3)
x.dir(mraa.DIR_OUT)
# x.period_us(700)
# x.enable(True)
# value=1;

while True:
    data = sys.stdin.readlines()
    # data =int(data)

    if data=='1':
	    x.write(1)
	    print('ON')
	    time.sleep(1)
	    x.write(0)
	    print('OFF')
	    time.sleep(1)

File: blink.js
var m = require('mraa'); //require mraa
console.log('MRAA Version: ' + m.getVersion());

var led = new m.Gpio(3); 
led.dir(m.DIR_OUT);
//var ledState = true; 


var blinkState = 0; 

function blink(){
  led.write(blinkState);
  blinkState = blinkState?0:1;   // toggle state
  //setInterval(blink,100); won't work

}

setInterval(blink,100);

